tests/test_protocol_coverage.py

```
@@ -0,0 +1,373 @@
+        backend.write({"test": 1})
+        backend.flush()
+        backend.close()
+
+        # Async call
+        async def test_async_method():
+            await backend.awrite({"async_test": 1})
+
+        asyncio.run(test_async_method())
+
+        # Verify all methods were called
+        expected_calls = [
```

Ruff (0.11.9)

Local variable `expected_calls` is assigned to but never used

Remove assignment to unused variable `expected_calls`

(F841)